tests/network/localnet/liblocalnet.py

```diff
@@ -8,7 +8,8 @@
 from tests.network.libs import cluster_user_defined_network as libcudn
 from tests.network.libs.label_selector import LabelSelector
 
-NETWORK_NAME = "localnet-network"
+LOCALNET_BR_EX_NETWORK = "localnet-br-ex-network"
```

Comment:
Why move them here? they are not used in this module.

Reply:
See [here](https://github.com/RedHatQE/openshift-virtualization-tests/pull/998/commits/90e04ed223972e628e46edc580437ebcb50d2bd5#diff-7294203e4291bdb425fa6b8446315f74806cc50c4c880dfee7ac9b782a15e62c).
For consistency, both are moved.

Reply:
Why not import them from tests/network/localnet/conftest.py? I don't see the advantage of having them here, where they are not used, and importing them in two modules. What am I missing?

Reply:
Because you cannot import from conftest.